src/In.ProjectEKA.DefaultHip/Link/PatientVerification.cs

```diff
@@ -24,15 +24,17 @@ public PatientVerification(IConfiguration configuration, HttpClient httpClient)
         
         public async Task<OtpMessage> SendTokenFor(Session session)
         {
-            var uri = new Uri(configuration.GetConnectionString("OTPGenerationConnection"));
+            var urlConfig = configuration.GetValue<string>("OtpService:BaseUrl");
```

Comment:
https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options?view=aspnetcore-3.1 Could you please check does it make sense to you use options pattern here?

Reply:
I added it earlier using a class but was not able to test it.